app/controllers/registrations_controller.rb

```diff
@@ -43,10 +43,12 @@ def autocomplete
   def source_id
```

Comment:
if this is a boolean method, should it be `source_id_unique?`

Reply:
This is not a boolean method, it's returning json.